### ISSUE TITLE
Install Macaulay2 in the image

### ISSUE THREAD

**Issue by zickgraf** [gap-system/gap-docker-base#23]
I would like to include [Macaulay2](http://www2.macaulay2.com/Macaulay2/) in the image, since this CAS can be used as a backend by `RingsForHomalg`. Would this be accepted? If yes, what would be the preferred way of installation: compiling from source or installing the official .deb package provided [here](http://www2.macaulay2.com/Macaulay2/Downloads/GNU-Linux/Ubuntu/index.html)?

**Comment by zickgraf** [comment]
Installing the .deb package would need about 60-70MB of additional disk space.

**Comment by olexandr-konovalov** [comment]
Might be ok. I see that "The RingsForHomalg package provides small dictionaries for homalg to speak (as much as needed of) the languages of Singular, Macaulay2, MAGMA, Sage, and Maple" - do you know any cases when Macaulay2 is preferable over Singular?

**Comment by zickgraf** [comment]
In the context of `RingsForHomalg`, Macaulay2 is useful for comparing results to Singular and finding bugs. In a broader context it has many features not available in Singular.

**Comment by olexandr-konovalov** [comment]
Ok, sounds like a good reason to add it. I don't mind either compiling from source or installing the official .deb package, if you are tolerating the fact that .deb packages may not always give the latest official Macaulay2 release. Would you like to submit a PR?

**Comment by zickgraf** [comment]
Yes, I would like to submit a PR. However, while doing so I noticed that it would actually require 195 MB of additional disk space. I thought the .deb provided on the download page would be a standalone package, but it depends on `macaulay2-common` which itself requires >100 MB of disk space. Sorry for this, would a PR still be accepted?

Edit: I also tried to compile from source but this first compiles lots of dependencies which takes time and space, so I think this is not a feasible approach.
